### PR TITLE
feat: port cacerts rule

### PR DESCRIPTION
**Pull request by thesayyn**
cc @alexeagle

### CODE REVIEW

File: distroless/private/cacerts.bzl [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            cacerts.path,
+            copyright.path,
+        ],
+    )
+
+    output = ctx.actions.declare_file(ctx.attr.name + ".tar.gz")
+    mtree = tar_lib.create_mtree(ctx)
+    mtree.add_file_with_parents("/etc/ssl/certs/ca-certificates.crt", cacerts)
+    mtree.add_file_with_parents("/usr/share/doc/ca-certificates/copyright", copyright)
+    mtree.build(output = output, mnemonic = "CaCertsTarGz", inputs = [cacerts, copyright])
```
Comment (thesayyn) [inline distroless/private/cacerts.bzl:58]: @alexeagle this is awesome. gives me more confidence that bsdtar was a good choice.

File: WORKSPACE.bazel [hunk 2]
```diff
@@ -1 +1,12 @@
 # Marker that this is the root of a Bazel workspace.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```
Comment (loosebazooka) [inline WORKSPACE.bazel:5]: Maybe I don't understand bazel, but should this be here? should examples have it's own workspace?

**Review by loosebazooka — APPROVED** [review]
looks good

File: WORKSPACE.bazel (hunk 2 above)
Comment (thesayyn) [reply WORKSPACE.bazel:5]: unfortunately yes, module.bazel doesn't have an easy way to use http_archive in module.bazel files.

Comment (thesayyn) [reply WORKSPACE.bazel:5]: this not necessary in bazel 7 but we are testing against 6 as well. so we are stuck using this

**Pull request merged**